snapshot-labs/snapshot
# feat: Add search for votes

Closes: https://github.com/snapshot-labs/snapshot/issues/3519

- [x] SpaceProposalVotesModal in SpaceProposalPage
- [x] Infinite scroll with useScroll from vueuse
- [x] Search with ETH and ENS addresses

## Changes
Added maxWidth prop to BaseModal
Added autofocus prop to BaseSearch

## How to test
Open any proposal page and click "Search" or "See more"

![image](https://user-images.githubusercontent.com/97898172/225281562-f3ccdc8c-1db5-404b-948c-9174649853de.png)
![image](https://user-images.githubusercontent.com/97898172/225281629-e4aa3eca-bec3-4434-b7f6-e328ac7d56eb.png)

## TBD
Its hard to click "See more" at the end of 10 votes on proposal, so I added "Search" on the top of Votes block
